OpenRA.Mods.Common/Traits/Production.cs

```diff
@@ -118,17 +118,11 @@ public virtual bool Produce(Actor self, ActorInfo producee, string factionVarian
 			if (Reservable.IsReserved(self))
 				return false;
 
-			if (!occupiesSpace)
```

Comment:
I put this here for a reason. It is the `CanUseExit()` below that crashes, so moving this check won't work.